### PR TITLE
Add voting status for participants

### PR DESCRIPTION
**Pull request by ElviaBth**
🎩 What? Why?
Add voting status for participants for each question.

📌 Related Issues
#175 
#174 Enlarge delegation button

📷 Screenshots
 - [x] If the user has not answered all the questions, should look "pending":
image

  - [x] If the user has answered all questions, should look "success":
image

- [x] When clicking on the message, a modal window should appear with a summary of all the questions and it's current status:
image
This modal is have a link to any non-answered question
- [x]  Create a simple system test to check the modal is opened

- [x] Delegation button current status before vote:
image

- [x] Delegation button current status after vote:
image

♥️ Thank you!

### CODE REVIEW

File: app/views/decidim/consultations/question_votes/_callout.html.erb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+          </thead>
+          <tbody>
+            <% current_question.consultation.questions.each do |question| %>
+              <tr>
+                <td><%= translated_attribute(question.title) %></td>
+                <td>
+                  <% if question.votes.where(author: current_user).any? %>
+                    <% user_response = question.responses.joins(:votes).find_by(votes: { author: current_user }) %>
+                      <% if user_response %>
+                        <p><%= translated_attribute(user_response.title) %></p>
```
Comment (microstudi) [inline app/views/decidim/consultations/question_votes/_callout.html.erb:33]:
```suggestion
                        <p title="<%= translated_attribute(user_response.title) %>"><%= t("booleans.true")%></p>
```

**Review by microstudi — CHANGES_REQUESTED** [review]
Nice job Elvia,
we need to check for the multioption responses as well.
Left you a couple of comments

File: app/views/decidim/action_delegator/consultations/questions/_link_to_delegations.html.erb [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+        <%= t("action_delegator.delegations.has_delegations", scope: "decidim") %>
+      </button>
     </div>
   <% else %>
-    <div class="delegations-notice flex--cc">
-      <%= link_to t("action_delegator.delegations.link", scope: "decidim"), "#", id: "delegations-button" %>
-    </div>
+    <div class="delegations-notice flex--cc" style="margin-top:1rem; width: 100%;">
+      <button id="delegations-button" class="card__button button small expanded warning" style="color:white;">
+        👉
```
[the remark this answers is not in the document]
Comment (ElviaBth) [reply app/views/decidim/action_delegator/consultations/questions/_link_to_delegations.html.erb:11]: Done

File: app/views/decidim/consultations/question_votes/_callout.html.erb (hunk 1 above)
Comment (ElviaBth) [reply app/views/decidim/consultations/question_votes/_callout.html.erb:33]: Thanks, done.

File: app/views/decidim/consultations/question_votes/_callout.html.erb [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<% total_published_questions = consultation.questions.published.count %>
+<% voted_questions = consultation.questions.published.joins(:votes).where(votes: { author: current_user }).distinct.count(:id) %>
+
+<% alert_class = total_published_questions != voted_questions ? "alert" : "success" %>
```
Comment (microstudi) [inline app/views/decidim/consultations/question_votes/_callout.html.erb:4]:
```suggestion
<% alert_class = total_published_questions != voted_questions ? "warning" : "success" %>
```

**Pull request merged**